docs/_docs/framework/capabilities/manage/policy.md

```diff
@@ -84,8 +84,8 @@ This capability is a part of the FinOps Framework by the FinOps Foundation, a no
 
 You can also find related videos on the FinOps Foundation YouTube channel:
 
-<!--[!VIDEO https://www.youtube.com/embed/{id}?list={list}]-->
-{% include video.html title="Cloud policy and governance videos" id="wxYMbhhA51dZNrky" list="PLUSCToibAswk2tx4PuWlztgSK9DZpL04b" %}
+<!--[!VIDEO https://www.youtube.com/embed/wiqCovCttOc?list=PLUSCToibAswnr2q_J-kn7Yii1CO2-PU35&pp=iAQB)]-->
```

Comment:
Didn't you want to add this video instead? https://www.youtube.com/embed/iL7H6oyzf7M?list=PLUSCToibAswk2tx4PuWlztgSK9DZpL04b&pp=iAQB

Reply:
That link would be for onboarding workloads. The one I have is for governance, so we should be good. I did add an extra parentheses, tho.
```suggestion
<!--[!VIDEO https://www.youtube.com/embed/wiqCovCttOc?list=PLUSCToibAswnr2q_J-kn7Yii1CO2-PU35&pp=iAQB]-->
```